src/controller/request/product-request.ts

```diff
@@ -43,6 +46,9 @@ export type ProductRequest = UpdateProductParams | CreateProductParams;
  * @property {number} vat.required - VAT group ID of the product
  * @property {number} category.required  - Category of the product
  * @property {number} alcoholPercentage.required  - Alcohol percentage of the product in 2 decimals
+ * @property {boolean} featured.required - If product is featured
```

Comment:
Why required? I would have them all be optional.

Reply:
Problem here is that UpdateProductParams and CreateProductParams are different. For creation, these params are required, for updating they are not.